components/CategorySelection/index.js

```diff
@@ -27,9 +28,9 @@ const CategoriesSelection = () => {
             <Card
               key={id}
               id={id}
-              title={name}
+              title={convertSlugToTitle(name)}
```

Comment:
The name of this function is way better. You can probably guess my next question now. If the function converts a slug to a title, why are you passing it the name and not the slug here?

Reply:
It looks to me that all the other calls to this function are called with a slug

Reply:
Makes perfect sense,  Done :)